.pre-commit-config.yaml

```
@@ -33,5 +33,7 @@ repos:
   - repo: https://github.com/codespell-project/codespell
     rev: "v2.4.1"
     hooks:
-    - id: codespell
-      args: ["-Ldynamc,notin"]
+      - id: codespell
+        additional_dependencies:
+          - tomli  # for python_version < '3.11'
```

```suggestion
          - "tomli; python_version<'3.11'"
```

95% sure this works, this is just passed on to the install tool.